xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd

```diff
@@ -0,0 +1,357 @@
+						<xsd:annotation>
+							<xsd:documentation>Identifier of ENTITY.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:attribute>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:element name="SpotEquipment" substitutionGroup="InstalledEquipment">
+		<xsd:annotation>
+			<xsd:documentation>An abstract EQUIPMENT in a LOCATABLE SPOT providing a onboard seat, bed or other amenity. +V2.0</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:restriction base="SpotEquipment_VersionStructure">
+					<xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="EntityInVersionGroup" minOccurs="0"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="DataManagedObjectGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="EquipmentGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="SpotEquipmentGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="SpotEquipmentIdType" use="optional">
+						<xsd:annotation>
+							<xsd:documentation>Identifier of ENTITY.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:attribute>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="SpotEquipment_VersionStructure" abstract="false">
+		<xsd:annotation>
+			<xsd:documentation>Type for a SPOT EQUIPMENT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="InstalledEquipment_VersionStructure">
+				<xsd:sequence>
+					<xsd:group ref="SpotEquipmentGroup"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="SpotEquipmentGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for SPOT EQUIPMENT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="Width" type="LengthType" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Width of seat bottom.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="HeightFromFloor" type="LengthType" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Height from bloor</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="PowerSupply" type="xsd:boolean" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether spot has power supply socket.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="UsbPowerSocket" type="xsd:boolean" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether Spot has UsbPowerSockets.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+		</xsd:sequence>
+	</xsd:group>
+	<!-- === SEAT EQUIPMENT ============================================ -->
+	<xsd:element name="SeatEquipment" substitutionGroup="InstalledEquipment">
```

Comment:
As we already have a "SeatingEquipment" (with a different purpose) with attributes like _Armrest_ and _SeatHeight_ , shouldn't we harmonise and also clarify when to use each other in the annotation ?